sympy/printing/pretty/pretty_symbology.py

```diff
@@ -449,21 +449,22 @@ def xsym(sym):
 # SYMBOLS
 
 atoms_table = {
-    # class             how-to-display
-    'Exp1':             U('SCRIPT SMALL E'),
-    'Pi':               U('GREEK SMALL LETTER PI'),
-    'Infinity':         U('INFINITY'),
-    'NegativeInfinity': U('INFINITY') and ('-' + U('INFINITY')),  # XXX what to do here
-    #'ImaginaryUnit':    U('GREEK SMALL LETTER IOTA'),
-    #'ImaginaryUnit':    U('MATHEMATICAL ITALIC SMALL I'),
-    'ImaginaryUnit':    U('DOUBLE-STRUCK ITALIC SMALL I'),
-    'EmptySet':         U('EMPTY SET'),
-    'Naturals':         U('DOUBLE-STRUCK CAPITAL N'),
-    'Integers':         U('DOUBLE-STRUCK CAPITAL Z'),
-    'Reals':            U('DOUBLE-STRUCK CAPITAL R'),
-    'Union':            U('UNION'),
-    'Intersection':     U('INTERSECTION'),
-    'Ring':             U('RING OPERATOR')
+    # class                    how-to-display
+    'Exp1':                    U('SCRIPT SMALL E'),
```

Comment:
I'm not sure about the role `atoms_table` play in pretty printing. Try reverting these changes and see if you still get what you wanted to get? I didn't have to touch this dict while adding printer for the Set Difference class. See https://github.com/hargup/sympy/commit/324aa9a5ec016dd1cb5c2e1e6b94aff27036049c